cypress/integration/details-test.js

```diff
@@ -0,0 +1,52 @@
+describe('', () => {
+  beforeEach(() => {
+    cy
+    .intercept('POST', 'https://pure-hollows-05817.herokuapp.com/https://tranquil-depths-91575.herokuapp.com/graphql', (req) => {
+      if (req.body.query.includes('The Payback')) {
+        req.reply({ fixture: 'details-fixture.json' })
+      }
+    })
+    .visit('http://localhost:3000')
+  })
+
+  it('should display the current album cover', () => {
+    cy.get('[data-cy=album-cover')
+      .should('have.attr', 'src', 'https://img.discogs.com/MUELn9ObTL-ZpxyUgF5M9D_Kumc=/fit-in/600x591/filters:strip_icc():format(jpeg):mode_rgb():quality(90)/discogs-images/R-719877-1247870750.jpeg.jpg')
+  });
+
+  it('should allow users to add the current album to their collection of favorites', () => {
+    cy.get('[data-cy=favorites-button')
+      .should('exist')
+  });
+
+  it('should allow users to visit discogs.com from the modal', () => {
+    cy.get('[data-cy=discogs-link')
+      .should('exist')
+  });
```

Comment:
Might be just these two that will get expanded on later, I'm tempted to suggest just the re-naming of these tests to 'render xxx' or 'xxx exist'

Reply:
As you alluded to, the intent was/is to expand on these later once functionality is added to these elements.  Whether those behavioral tests replace the above or just supplement the existence tests, I'm content.